.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
Is this supposed to be committed? How does husky make this cross platform?

Reply:
but yeah that's how it works, i think the precommit script is supposed to be commited